soroban-spec-rust/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2021"
 rust-version.workspace = true
 
 [dependencies]
-stellar-xdr = { workspace = true, features = ["curr", "std", "serde"] }
+stellar-xdr = { workspace = true, features = ["next", "std", "serde"] }
```

Comment:
This is surprising, why do we need next for the specs? The specs would have been updated in both?